release-tools/.prow.sh

```diff
@@ -1,7 +1,23 @@
 #! /bin/bash -e
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 # This is for testing csi-release-tools itself in Prow. All other
 # repos use prow.sh for that, but as csi-release-tools isn't a normal
 # repo with some Go code in it, it has a custom Prow test script.
 
 ./verify-shellcheck.sh "$(pwd)"
+./verify-spelling.sh "$(pwd)"
```

Comment:
Is the goal to also enable these checks in the nfs repo? If so, we also need to add these calls to this repo's .prow.sh

Reply:
> Is the goal to also enable these checks in the nfs repo?

Yes, but this PR is solely to update the release tools for nfs repo. I intend to make a separate PR for enabling these tests for the prow and in the makefile (by enabling them in `hack/verify-all.sh`) hence fixing #128. Is it okay?

Reply:
sounds good thanks!